doc/BUILD.md

```diff
@@ -1,6 +1,71 @@
 Build Instructions
 ==================
 
+## 1. Platform
+* Ubuntu 18.04
+
+## 2. Build Steps
+### 2.1 ROS2
+* Install ROS2 with dependencies from the ROS2 Installation Page: https://index.ros.org/doc/ros2/Installation/
+* Install the latest distribution of ROS2 to support Navigation2 master.
+* ROS2 can be installed either from binary packages or build from source. Choose to build ROS2 from source to support Navigation2 master.
+
+### 2.2 Navigation2 Dependencies
```

Comment:
Good instructions, but why not point to the `init_workspace.sh` script? I think this is all handled in that

Reply:
Perhaps just a matter of preference. Breakdown of instructions is helpful when you do not have the need to run them all or when there is a failure in one of the steps that needs to be addressed before running other steps.

Reply:
Sorry, what I meant was, can you also post a blurb saying that 'hey this is all in this script over here' so that they can just use it if they want?